### DISCUSSION TITLE
Delete invoices?

### DISCUSSION THREAD

**Discussion by tannenfreund87** [dragotin/kraft#220] [Q&A] [answered]
I'm trying to set up Kraft for my needs, especially using the X-Rechnung funtionality. Doing so, I created some exemplary invoices, modelled after invoices I already sent in the past. Now, these invoices are bogus, as they were just for training, but I cannot find a way to delete them. Do I need to start all over with a new database?

**Comment by dragotin** [comment, marked as answer]
Starting over with a new database is save and comparably easy. You could also delete the docs directly from the database, using a tool like sqlitebrowser, if you have little insight into SQL databases. 

In Kraft 2.0 that will be more easy.
